docs/attributes.md

```diff
@@ -92,6 +92,14 @@ axiom{R} \equals{SortSet{}, R} (
 productionID{}("id")
 ```
 
+### Priority
+
+```
+priority{}("123")
+```
+
+The priority attribute specifies a number which determines the order of rule application. The attribute's argument is a string literal representing a decimal integer.
```

Comment:
We usually keep the line length below 80 characters unless there is a good reason to make it longer in order to accommodate more editor/IDE setups.